File: 프로그래머스/lv1/12934. 정수 제곱근 판별/정수 제곱근 판별.py
def solution(n):
    answer = -1             # 어떤 수의 제곱이 아니라면 -1 반환
    
    for i in range(1, n+1):
        if i ** 2 == n:
            answer = (i+1) ** 2
            break
    
    return answer

File: 프로그래머스/1/12934. 정수 제곱근 판별/정수 제곱근 판별.java
class Solution {
    public long solution(long n) {
        long answer = -1;
        int i = 1;
        long value = 0;
        
        while (value < n) {
            value = (long)Math.pow(i, 2);
            i++;
            if (value == n) {
                answer = (long)Math.pow(i, 2);             
                break;
            }              
            
        }
        return answer;
    }
}